index.php

```diff
@@ -2,7 +2,7 @@
     include("styla.php");
 
     $styla = new Styla('showcase', [
-        "rootpath" => "/"
+        "rootPath" => "/"
```

Comment:
1. first issue was that the array key is case senstive, so the styla.php was receiving a NULL string